src/object.d

```diff
@@ -3394,7 +3404,7 @@ private void _doPostblit(T)(T[] arr)
             S3[] m;
             immutable(S3)[] i;
             static assert(!is(typeof(m.dup)));
-            static assert(!is(typeof(i.idup)));
+            static assert( is(typeof(i.idup)));
```

Comment:
:warning: Changing an existing test here. Arrays of non-copyable immutable objects are now "copyable" in the sense that `.idup` succeeds. In theory, this is perfectly OK because of the transitivity of D's constness.